apps/jira/jira-app/src/components/Jira/Search.tsx

```diff
@@ -27,7 +27,7 @@ export default class Search extends React.Component<Props, State> {
   async componentDidUpdate(_prevProps: Props, prevState: State) {
     const { value } = this.state;
 
-    if (value.length > 4 && value !== prevState.value) {
+    if (value.length > 2 && value !== prevState.value) {
```

Comment:
Any reason why start a search with fewer keystrokes? Maybe a debounce of sorts in case they're still typing?